services/pypi/pypi-helpers.spec.js

```diff
@@ -100,6 +101,29 @@ describe('PyPI helpers', function() {
     ])
   })
 
+  test(getLicenses, () => {
+    forCases([given({ info: { license: 'MIT', classifiers: [] } })]).expect([
+      'MIT',
+    ])
+    forCases([
+      given({
+        info: {
+          license: '',
+          classifiers: ['License :: OSI Approved :: MIT License'],
+        },
+      }),
+      given({
+        info: {
+          license: '',
+          classifiers: [
+            'License :: OSI Approved :: MIT License',
+            'License :: DFSG approved',
+          ],
+        },
+      }),
+    ]).expect(['mit license'])
```

Comment:
Could we have an example for the acronym case? I assume this is for eg. [magma](https://pypi.org/pypi/magma/json)?